src/main/groovy/io/seqera/wave/service/logs/BuildLogServiceImpl.groovy

```diff
@@ -69,34 +73,41 @@ class BuildLogServiceImpl implements BuildLogService {
     @Value('${wave.build.logs.maxLength:100000}')
     private long maxLength
 
+    @Nullable
+    @Value('${wave.build.logs.conda-lock-prefix}')
+    private String condaLockPrefix
+
     @Inject
     @Named(TaskExecutors.IO)
     private volatile ExecutorService ioExecutor
 
     @PostConstruct
     private void init() {
-        log.info "Creating Build log service bucket=$bucket; prefix=$prefix; maxLength: ${maxLength}"
+        log.info "Creating Build log service bucket=$bucket; logs prefix=$prefix; maxLength: ${maxLength}; condaLock prefix=$condaLockPrefix"
     }
 
     protected String logKey(String buildId) {
         if( !buildId )
             return null
         if( !prefix )
             return buildId + '.log'
-        final base = org.apache.commons.lang3.StringUtils.strip(prefix, '/')
+        final base = strip(prefix, '/')
         return "${base}/${buildId}.log"
     }
 
     @EventListener
     void onBuildEvent(BuildEvent event) {
         if(event.result.logs) {
             CompletableFuture.supplyAsync(() -> storeLog(event.result.id, event.result.logs), ioExecutor)
+            CompletableFuture.supplyAsync(() -> storeCondaLock(event.result.id, event.result.logs), ioExecutor)
```

Comment:
I think there's no need to expose `storeCondaLock` independently, the storing of the lock file can stay in  `storeLog` implementation since it's a byproduct of the logs

Reply:
done